workflows/example_single_family_detached_TMY.osw

```diff
@@ -248,8 +247,7 @@
         "weekday_setpoint": "76.0",
         "weekend_offset_magnitude": "0",
         "weekend_offset_schedule": "0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0",
-        "weekend_setpoint": "76.0",
-        "onoff_thermostat_deadband": "0"
```

Comment:
@jmaguire1 It seemed these example osws already auto-updated(not covered by my manual changes, sorry for misleading) their arguments to include the new thermostat deadband (default to 0).